tests/utils.py

```diff
@@ -46,3 +47,39 @@ def create_test_yaml(fg=4, bg=2):
     with open('/tmp/secrets.yml', 'w+') as fh:
         fh.write(yaml.dump(data))
     return '/tmp/'
+
+
+# def create_test_kubeconfig():
```

Comment:
Killing this? Keeping it?

Reply:
I want to use it, but the way I've defined the reading from kubeconfig currently (for cluster name, in init) means i can't get this function to run early enough during tests. :D
Might change that later..
I think we'll still want to use this dummy kube config when we add more tests in future.